tools/setup-envtest/version/version.go

```diff
@@ -0,0 +1,21 @@
+package version
+
+import "runtime/debug"
+
+// Version to be set using ldflags:
+// -ldflags "-X sigs.k8s.io/controller-tools/pkg/version.version=v1.0.0"
```

Comment:
```suggestion
// -ldflags "-X sigs.k8s.io/controller-runtime/tools/setup-envtest/version.version=v1.0.0"

````

Reply:
@sbueringer I can make a PR for this to fix the godoc rendering with the correct comment.

Reply:
Ups. Yup, thx!